bin/adam-submit

```diff
@@ -52,15 +52,29 @@ fi
 # Figure out where ADAM is installed
 SCRIPT_DIR="$(cd `dirname $0`/..; pwd)"
 
-# Get list of required jars for ADAM
-ADAM_JARS=$("$SCRIPT_DIR"/bin/compute-adam-jars.sh)
+# Find ADAM cli assembly jar
+ADAM_CLI_JAR=
+if [ -f "$SCRIPT_DIR/repo" ]; then
+  ASSEMBLY_DIR="$SCRIPT_DIR/repo"
+else
+  ASSEMBLY_DIR="$SCRIPT_DIR/adam-cli/target"
+fi
 
-# Split out the CLI jar, since it will be passed to Spark as the "primary resource".
-ADAM_CLI_JAR=${ADAM_JARS##*,}
-ADAM_JARS=$(echo "$ADAM_JARS" | rev | cut -d',' -f2- | rev)
+num_jars="$(ls -1 "$ASSEMBLY_DIR" | grep "^adam-cli.*\.jar$" | wc -l)"
```

Comment:
This bit also needs to be more clever, avoiding other artifacts such as adam-cli_...-javadoc.jar
```bash
$ ./bin/adam-shell -h
Found multiple ADAM cli assembly jars in /Users/heuermh/working/adam/adam-cli/target:
adam-cli_2.10-0.17.2-SNAPSHOT-javadoc.jar
adam-cli_2.10-0.17.2-SNAPSHOT.jar
Please remove all but one jar.
```